devtools/conda-recipe/meta.yaml

```diff
@@ -33,12 +33,13 @@ test:
     - ipyext
 
   source_files:
-    - mbuild/examples/*
-
+    - .coveragerc
+  
   commands:
-    - py.test -v --pyargs mbuild
-    - cd mbuild/examples
-    - py.test test_examples.py -v
+    - conda install -yc conda-forge python-coveralls 
+    - conda install -yc conda-forge pytest-cov
```

Comment:
I think this is causing some versioning [issues on windows](https://ci.appveyor.com/project/ctk3b/mbuild-o0viu/build/1.0.366/job/2m1mdic8ppmg41a7).

In general I would not use full installation commands in the meta.yml. It __should__ work just by adding them to the `requires` list but you said this causes a failed install?